kolena/dataset/__init__.py

```diff
@@ -16,12 +16,15 @@
 from kolena.dataset.dataset import fetch_dataset
 from kolena.dataset.dataset import register_dataset
 from kolena.dataset.evaluation import fetch_results
-from kolena.dataset.evaluation import test
+from kolena.dataset.evaluation import upload_results
+from kolena.workflow import annotation, asset
```

Comment:
need to remove this (have you rebased against latest trunk)?

Reply:
I took the wrong diff when I fixed the merge conflict earlier, good catch